community/urlview/build

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh -e
+
+gunzip urlview_0.9-21.diff.gz
+patch -p1 < urlview_0.9-21.diff
+
+# Move user config to ~/.config/urlview/conf
+sed -i 's|%s\/\.urlview|%s\/\.config/urlview/conf|' urlview.c
+sed -i 's|~/.urlview|~/.config/urlview/conf|g' urlview.man
+
+# The included url_handler.sh is extremely outdated -- use $BROWSER by default
+# We don't want $BROWSER to expand, so
+# shellcheck disable=SC2016
+sed -i 's|COMMAND url_handler\.sh|COMMAND \$BROWSER %s|' sample.urlview
+
+aclocal
+automake \
+    --foreign \
+    --ignore-deps \
+    --add-missing
```

Comment:
Why? No pre-generated files?

Reply:
The original sources assumes assume automake-1.11 and the debian patch is quite substantial. The build fails without this..

Reply:
Why not just call `autoreconf` (may need `-i` or `-fis`)?

Reply:
That unfortunately isn't sufficient. I can only get it to build with these lines.